acoustid/web/views/apps.py

```
@@ -23,36 +24,40 @@
     user = require_user()
     title = "Your Application"
-    conn = db.session.connection()
+    conn = db.get_app_db()
     application = conn.execute(
-        """
+        sql.text(
+            """
         SELECT * FROM application
         WHERE id = %s
-    """,
+    """
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)